telemetry/telemetry.go

```diff
@@ -72,6 +85,46 @@ func runTelemetry(args []string) error {
 	return nil
 }
 
+func getGlogPrefixes() []string {
+	// glog flags: https://pkg.go.dev/github.com/golang/glog
+	return []string {
+		"-alsologtostderr",
+		"-log_backtrace_at",
+		"-log_dir",
+		"-log_link",
+		"-logbuflevel",
+		"-logtostderr",
+		"-stderrthreshold",
+		"-v",
+		"-vmodule",
+	}
+}
+
+func parseOSArgs() ([]string, []string) {
+	glogFlags := []string{os.Args[0]}
+	telemetryFlags := []string{os.Args[0]}
+
+	for _, arg := range os.Args[1:] {
+		if strings.HasPrefix(arg, "-v") { // only flag in both glog and telemetry
+			glogFlags = append(glogFlags, arg)
+			telemetryFlags = append(telemetryFlags, arg)
+			continue
+		}
+		isGlogFlag := false
+		for _, prefix := range getGlogPrefixes() {
```

Comment:
Can you use hashmap to avoid iterating list?

Reply:
Done